rss2017/process/processToFile.js

```diff
@@ -0,0 +1,41 @@
+var fs = require('fs');
+
+var mainProcessFile = process.argv[2];
+var outputPath = process.argv[3];
+var track = process.argv[4];
+
+console.log('Main process', mainProcessFile);
+
+var mainProcess = fs.readFileSync(mainProcessFile, 'utf8');
+
+// Look for << filename.js>> and replace that string with the code in the file
+var regex = new RegExp('<<(.+\.js)>>', 'g');
+var matches = mainProcess.match(regex);
+console.log('matches', matches);
+if(matches) {
+
+	matches.forEach(function(m) {
+
+		var fileName = m.replace('<<', '').replace('>>', '');
+		var fileContent = fs.readFileSync(outputPath + '/' + fileName, 'utf8');
+		mainProcess = mainProcess.replace(m, fileContent);
+		console.log('Processing file', fileName);
+	})
+}
+
+
+// If there is track information add it as an argument to the function call
+if (track) {
```

Comment:
Please, avoid creation of a new script: rss2017/process/processToFile.js

try to solve it so we can use it in all the conferences. One possible solution could be:

```
function() {
   var or3client = lib.or3client;
   const TRACK_NAME = <<param1>>
   const CONFERENCE = 'roboticsfoundation.org/RSS/2017/Workshop';
   const COCHAIRS = CONFERENCE+"/Program_Co-Chairs"
   const TRACK = CONFERENCE+'/'+TRACK_NAME
   const PAPERGRP = TRACK+'/Paper' + note.number;
   const PAPERINV = TRACK+'/-/Paper' + note.number;
   const DUE_DATE = new Date(2017, 6, 30, 17, 15);

```

and then if the processToFile receives more than 3 arguments then find and replace:

```
mainProcess.replace('<<param1>>', process.argv[4])
```

in general

```
mainProcess.replace('<<param' + i + '>>', process.argv[3 + i])
```